core/app/models/spree/reimbursement_tax_calculator.rb

```diff
@@ -23,18 +23,9 @@ def call(reimbursement)
       def set_tax!(return_item)
         calculated_refund = Spree::ReturnItem.refund_amount_calculator.new.compute(return_item)
 
-        percent_of_tax = if return_item.pre_tax_amount <= 0 || calculated_refund <= 0
-          0
-        else
-          return_item.pre_tax_amount / calculated_refund
-        end
-
-        additional_tax_total = percent_of_tax * return_item.inventory_unit.additional_tax_total
-        included_tax_total   = percent_of_tax * return_item.inventory_unit.included_tax_total
-
         return_item.update_attributes!({
-          additional_tax_total: additional_tax_total,
-          included_tax_total:   included_tax_total,
+          additional_tax_total: return_item.inventory_unit.additional_tax_total,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.